docs/adding-a-page.md

```diff
@@ -0,0 +1,34 @@
+## 📄 Adding a Page to Your Website
+
+Before you start, ensure you have generated a Strapi API token with both **write** and **read** access, and that you've added it to your `.env.local` file.
+
+### Step 1: Access the Strapi Dashboard 🔑
+
+1. Go to the **Content Manager** in your Strapi Dashboard. You will need admin permissions to access this feature.
+
+### Step 2: Select or Create a Collection Type 📚
+
+1. Select **Page** in the Collection Types. If you need to create a new collection type, you can do so in the **Content Type Builder**. [@todo Add link to Content Type Builder doc here].
```

Comment:
If you need to create a new collection type, you can do so in the **Content Type Builder**. [@todo Add link to Content Type Builder doc here]. -> Precise that it's not necessary, it should be already here